pom.xml

```diff
@@ -48,7 +48,7 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <java.version>1.8</java.version>
-        <tinkerpop.version>3.1.1-incubating</tinkerpop.version>
```

Comment:
Cool. But a nitpick for a future PR is to change 

```
<titan.version>1.1.0.grakn-SNAPSHOT</titan.version>
```

to 

```
<titan.version>1.1.0-grakn-SNAPSHOT</titan.version>
```

Or maybe just make that change when we migrate to Janus. Anyway, just noting it here.